installscripts/cookbooks/jenkins/recipes/sonarqube-scanner.rb

```diff
@@ -0,0 +1,53 @@
+remote_file "#{Chef::Config['file_cache_path']}/sonar-scanner-cli-3.0.3.778-linux.zip" do
+  source 'https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.0.3.778-linux.zip'
+  mode '0755'
+  action :create
+end
+
+execute 'install_sonarscanner' do
+  command "unzip -o #{Chef::Config['file_cache_path']}/sonar-scanner-cli-3.0.3.778-linux.zip"
+  cwd '/opt'
+end
+
+directory '/opt/sonar-scanner-3.0.3.778-linux/' do
+  owner 'jenkins'
+  group 'jenkins'
+  recursive true
+end
+
+link '/opt/sonar-scanner-3.0.3.778-linux/bin/sonar-scanner' do
+  to '/bin/sonar-scanner'
+end
+
+directory '/var/log/depcheck_jazz/nistfiles' do
+  owner 'jenkins'
+  group 'jenkins'
+  recursive true
+end
+
+remote_file "#{Chef::Config['file_cache_path']}/dependency-check-3.2.1-release.zip" do
+  source 'https://bintray.com/jeremy-long/owasp/download_file?file_path=dependency-check-3.2.1-release.zip'
+  mode '0755'
+  action :create
+end
+
+execute 'install_dependency_check' do
+  command "unzip -o #{Chef::Config['file_cache_path']}/dependency-check-3.2.1-release.zip"
+  cwd '/opt'
+end
+
+directory '/opt/depency-check' do
```

Comment:
Typo! It should be /opt/dependency-check

Reply:
@rajeevr2715 Ah shoot good catch. Sorry, I will catch up on some of this stuff today,